scripts/cli/TimerMode.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2022 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoQtiTest\scripts\cli;
+
+use oat\oatbox\extension\script\ScriptAction;
+use oat\oatbox\reporting\Report;
+
+class TimerMode extends ScriptAction
```

Comment:
What about?
```suggestion
class ChangeTimerMode extends ScriptAction
```

Reply:
Thanks @quintanilhar!
Initially I set a similar name, but then though it would be more generic since it also allows to know the current setting. Actually, I see it as a TimerMode manager tool. But if you think it is clearer this way, I can update it.

Reply:
Perhaps we can use `dry-run` for that. We already have the config itself to see what is the current value. Not sure why do we need this retrieving part and also it's not that to guess that. Ideally, action should do only a single task. We can create another one if really required. What do you think?

Reply:
Not sure a dry-run is needed for such a simple change. But why not.
The reason why I wanted to add a display option is because the timer config is lost in the middle of a big file.